data/channels.csv

```diff
@@ -7526,6 +7526,7 @@ GravitasAdrenaline.us,Gravitas Adrenaline,,,,US,,,c/US,eng,,FALSE,,,,https://wat
 GravitasDocumentaries.us,Gravitas Documentaries,,,,US,,,c/US,eng,,FALSE,,,,https://watch.plex.tv/live-tv/channel/gravitas-documentaries,https://i.imgur.com/g3UY941.png
 GravitasMovies.us,Gravitas Movies,,,,US,,,c/US,eng,movies,FALSE,,,,,https://i.imgur.com/H2hD4vT.png
 GreatAmericanCountry.us,Great American Country,,,,US,,,c/US,eng,,FALSE,,,,,https://i.imgur.com/TzvnHDn.png
+GreatAmericanLiving.us,Great American Living,,,,US,,,c/US,eng,,FALSE,,,,https://www.greatamericanliving.tv/,https://imgur.com/a/dRQMkv5

```

Comment:
```suggestion
GreatAmericanLiving.us,Great American Living,,,,US,,,c/US,eng,,FALSE,,,,https://www.greatamericanliving.tv/,https://i.imgur.com/ccBJuRA.png
```

Logo size reduced to 512px.

https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels